### PR TITLE
Connection setting handling

### PR DESCRIPTION
**Pull request by krlm**
This PR adds handling of connection setting of queue storage in a similar way as it's done by regular `QueueTrigger`.

### CODE REVIEW

File: src/QueueBatch.Tests/BaseTest.cs
Comment (Scooletz) [inline src/QueueBatch.Tests/BaseTest.cs]: `storageAccount` is never used again. Why it is extracted as a field? Previous version had it inlined.

File: src/QueueBatch/Impl/BindingProvider.cs
Comment (Scooletz) [inline src/QueueBatch/Impl/BindingProvider.cs]: `StorageAccountProvider` has a simple API that obtains an account from the passed name. Would you consider extracting it to `QueueBatch` instead of referencing `Microsoft.Azure.WebJobs.Extensions.Storage`  for this only? Have you considered it?

File: src/QueueBatch/Impl/BindingProvider.cs
Comment (Scooletz) [inline src/QueueBatch/Impl/BindingProvider.cs]: As far as I see, this adds a new functionality of using custom expressions for queue names. Is it required for handling different accounts, or maybe it's provided to mimic `Storage` behavior? If the second, I'd prefer to keep it out of this PR.

File: src/QueueBatch/QueueBatch.csproj
Comment (Scooletz) [inline src/QueueBatch/QueueBatch.csproj]: Mentioned in the comment above.

File: src/QueueBatch/QueueBatchTriggerAttribute.cs
Comment (Scooletz) [inline src/QueueBatch/QueueBatchTriggerAttribute.cs:54]: Would it be possible to document it in the Readme.md somehow? So far all the properties have their own sections.

**Review by Scooletz — CHANGES_REQUESTED** [review]
Awesome stuff 🥇 
If we could clarify a few items, it'd be great.

File: src/QueueBatch/Impl/BindingProvider.cs
Comment (krlm) [reply src/QueueBatch/Impl/BindingProvider.cs]: Yes, I've  noticed that class is quite small and thought about extracting it but wasn't sure if other stuff from `Storage.Extension` will be needed, so finally kept that dependency. However, I completely understand that you'd like to minimize dependency footprint. I'll rewrite it.

File: src/QueueBatch/Impl/BindingProvider.cs
Comment (krlm) [reply src/QueueBatch/Impl/BindingProvider.cs]: Yes, this just handles `binding expressions` for queue name. It's not necessary for handling separate storage but allows you to configure queue name from external configuration (file/env.var). Otherwise you're forced to hardcode queue name. (or there's another way?)

File: src/QueueBatch/QueueBatchTriggerAttribute.cs
Comment (krlm) [reply src/QueueBatch/QueueBatchTriggerAttribute.cs:54]: Sure.

File: src/QueueBatch.Tests/BaseTest.cs
Comment (krlm) [reply src/QueueBatch.Tests/BaseTest.cs]: My bad, didn't clean up properly.

File: src/QueueBatch/Impl/BindingProvider.cs
Comment (krlm) [reply src/QueueBatch/Impl/BindingProvider.cs]: I removed that change from here. Let me know if you'd consider accepting this feature as a separate PR.

File: src/QueueBatch/Impl/BindingProvider.cs
Comment (Scooletz) [reply src/QueueBatch/Impl/BindingProvider.cs]: Great 😍

File: src/QueueBatch/Impl/BindingProvider.cs
Comment (Scooletz) [reply src/QueueBatch/Impl/BindingProvider.cs]: Yes, I see the value of having it resolved, not hard-coded. Of course I'd be happy to accept it as a separate PR.

File: src/QueueBatch/QueueBatchTriggerAttribute.cs
Comment (Scooletz) [reply src/QueueBatch/QueueBatchTriggerAttribute.cs:54]: Resolved in the next commit.

**Comment by krlm** [comment]
@Scooletz I've fixed all of above issues. Am I supposed to mark those conversations as resolved or just wait for further review? (I'm not really familiar with github reviews workflow)

**Comment by Scooletz** [comment]
Nope. Let me review it tomorrow in the morning, rerun the tests, etc. Once I'm done, I'll merge it.

Thank you again for introducing this feature 🤗

**Comment by Scooletz** [comment]
@krlm I added a few minor fixes in here: https://github.com/krlm/QueueBatch/pull/1 If you could accept it (squash-merge) preferred, I'd accept this one. Maybe I could have propose all the changes in here, but it was easier for me to get this as a PR for your PR.

**Review by Scooletz — APPROVED** [review]

**Comment by Scooletz** [comment]
Thank you @krlm  for bringing it! 🤗

**Pull request merged**
